docs/source/conf.py

```diff
@@ -126,6 +126,9 @@ def previous_version(ver):
 .. _exchange: https://exchange.stackstorm.org/
 .. |bwc| replace:: Extreme Workflow Composer
 .. |ipf| replace:: IP Fabric Automation Suite
+.. |br| raw:: html
+
+   <br />
```

Comment:
What's the reasoning behind needing this substitution? You shouldn't really need to be inserting HTML line breaks like that.

Reply:
It's just a common set of shortcuts to re-use.
I think it might be a good instrument to have.

I'm using that `|br|` substitution in several places here: https://github.com/StackStorm/st2docs/blob/60148d18e413e6e15fd60f98974306cdb463fae2/docs/source/install/vagrant.rst